modules/ROOT/pages/promote-api.adoc

```diff
@@ -52,20 +52,12 @@ To promote an API:
 . Click *Next: Provide Authentication*.
 . On the *Provide authentication* page, select the type of authentication on the GET endpoints for the API that you want to add, allowing Anypoint DataGraph to make requests to your API when queried:
 +
---
 * *No Auth*: Use if your API is public.
 * *Basic Auth*: Use if you authenticate to your API using a username and password.
+* *Pass-through*: Use if you authenticate to your API by passing authorization headers.
```

Comment:
@mauro-dentoni this is for pass-through auth when you promote an api

Reply:
Looks fine